presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeAggregations.java

```diff
@@ -260,6 +260,22 @@ public void testArbitrary()
         assertQuerySucceeds("SELECT orderkey, arbitrary(tax_as_real) FROM lineitem GROUP BY 1");
     }
 
+    @Test
+    public void testBlackhole()
+    {
+        assertQuery("SELECT blackhole(v) FROM (VALUES 1.0, 3.0, 5.0, NULL ) as t (v)");
```

Comment:
can you also add tests for presto java to AbstractTestAggregations.  You'll probably have to provide a second query with the expected results since h2 (the comparison db) presumably doesn't have a blackhole function.  e.g. it might look like

`assertQuery("SELECT blackhole(orderkey) FROM lineitem", "SELECT 'OK' FROM lineitem");`